docker_deploy/config.go

```diff
@@ -50,7 +52,8 @@ func (conf *Config) HasChanged(newConf *Config) bool {
 	if conf.RunOptions != nil && newConf.RunOptions != nil {
 		return !StringSliceEqual(conf.RunOptions.Env, newConf.RunOptions.Env) ||
 			!StringSliceEqual(conf.RunOptions.EntryPointArgs, newConf.RunOptions.EntryPointArgs) ||
```

Comment:
[nit] either expose both `StringSliceEqual` and `mapsEqual` or expose neither

Reply:
we shouldn't expose either